### PR TITLE
timer in python

### PR DESCRIPTION
**Pull request by SeDe-cmd**
The timer uses the interface tkinter. With threading the application may use several more functions while the timer is running. This makes it possible to create more features such as pausing the timer or ingeneral having the application perform different tasks while the timer is running.

### CODE REVIEW

File: scripts/timer.py [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+
+error_label = tk.Label(root, text="Please enter a valid number", font=("Times",12,), fg="#f00")
+entry_label = tk.Label(root, text="Enter timer duration (seconds):", font=("Times",48))
+entry_label.grid(row=0, column=0, pady=10)
+entry = tk.Entry(root)
+entry.grid(row=1, column=0, pady=10)
+
+timer_label = tk.Label(root, text="", font=("Times",48))
+
+def display_timer(seconds):
```
Comment (GiovanniBruno22) [inline scripts/timer.py:17]: Trivial: Can we add docstrings to functions to document what happens and what they expect as input (if any) and return as output(if any)?
For example:
```
def display_timer(seconds):
    """ Display the timer.
         args: seconds (int)
    """
```

**Review by GiovanniBruno22 — CHANGES_REQUESTED** [review]

File: scripts/timer.py (hunk 1 above)
Comment (SeDe-cmd) [reply scripts/timer.py:17]: Is the new commits docstrings enough or do i need to go into more detail?

Comment (GiovanniBruno22) [reply scripts/timer.py:17]: It is perfect, thank you so much for adding them!

**Review by GiovanniBruno22 — APPROVED** [review]
Awesome work! thank you so much for this!

**Pull request merged**